src/kOS/Binding/FlightControlManager.cs

```diff
@@ -225,6 +227,15 @@ public void SelectAutopilotMode(VesselAutopilot.AutopilotMode autopilotMode)
             }
         }
 
+        public string GetAutopilotModeName()
+        {
+            // TODO: As of KSP 1.0.4, RadialIn and RadialOut are swapped.  Check if still true in future versions.
```

Comment:
I don't suppose you checked this value in KSP 1.1.2?  I seem to recall a mention in a change log about fixing the radial button, but that may have just meant that when they redid the Button UI they didn't reverse the two either.  The fix may have been to fix the enums, or it could have been to add back in the odd behavior.